server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailSubmissionSetMethodFutureReleaseContract.scala

```diff
@@ -64,6 +168,22 @@ trait EmailSubmissionSetMethodFutureReleaseContract {
 
   def randomMessageId: MessageId
 
+  @Test
+  @Tag(CategoryTags.BASIC_FEATURE)
+  def serverShouldBeAdvertisedFutureReleaseExtension(): Unit = {
+    val sessionJson: String = `given`()
+    .when()
+      .header(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER)
+      .get("/session")
+    .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+    assertThatJson(sessionJson).isEqualTo(future_release_session_object)
```

Comment:
is it necessary to check the all json? we could not just check the related fields that they advertize correctly the future release instead?

Reply:
shold be fine actually